app/src/main/java/com/digitalarchitects/rmc_app/screens/EditMyVehicleScreen.kt

```diff
@@ -0,0 +1,317 @@
+                        horizontalArrangement = Arrangement.spacedBy(dimensionResource(R.dimen.padding_large))
+                    ) {
+                        Column(Modifier.weight(1f)) {
+                            RmcFilterChip(
+                                label = stringResource(id = R.string.engine_type_ice),
+                                selected = uiState.engineType == EngineType.ICE,
+                                onClick = {
+                                    viewModel.onEvent(EditMyVehicleUIEvent.EngineTypeICEButtonClicked)
+                                }
+                            )
+                        }
+                        Column(Modifier.weight(1f)) {
+                            RmcFilterChip(
+                                label = stringResource(id = R.string.engine_type_bev),
+                                selected = uiState.engineType == EngineType.BEV,
+                                onClick = {
+                                    viewModel.onEvent(EditMyVehicleUIEvent.EngineTypeBEVButtonClicked)
+                                }
+                            )
+                        }
+                        Column(Modifier.weight(1f)) {
+                            RmcFilterChip(
+                                label = stringResource(id = R.string.engine_type_fcev),
+                                selected = uiState.engineType == EngineType.FCEV,
+                                onClick = {
+                                    viewModel.onEvent(EditMyVehicleUIEvent.EngineTypeFCEVButtonClicked)
+                                }
+                            )
+                        }
+                    }
+                    RmcSpacer(8)
+
+                    Divider()
+
+                    RmcSpacer(8)
+
+                    Row(
+                        modifier = Modifier
+                            .fillMaxWidth(),
+//                        horizontalArrangement = Arrangement.spacedBy(dimensionResource(R.dimen.padding_large)),
+                        horizontalArrangement = Arrangement.SpaceBetween,
+                        verticalAlignment = Alignment.CenterVertically
+                    ) {
+
+                        Text(
+                            text = stringResource(id = R.string.vehicle_availability),
+                            fontWeight = FontWeight.Bold
+                        )
+                        BooleanToggle(value = uiState.availability, onToggle = {
+                            viewModel.onEvent(EditMyVehicleUIEvent.AvailabilityToggleButtonClicked)
+                        })
+                    }
+
+                    RmcSpacer(8)
+
+
+                    Row(
+                        modifier = Modifier
+                            .fillMaxWidth(),
+                        horizontalArrangement = Arrangement.spacedBy(dimensionResource(R.dimen.padding_large))
+                    ) {
+                        RmcTextField(
```

Comment:
- Als je hier in typt crashed de app. Even kijken hoe we dit oplossen/afvangen.
- 0.0 blijft altijd staan in het tekstveld

Reply:
Sowieso moeten we nog oplossen dat we van een locatie een latlong maken. Of, voor nu, de locatie opvragen van de gebruiker (via de Maps API). Ik was nog van plan om locatievoorziening toe te voegen aan de map. Dus later...

Reply:
ja even op de todo zetten. Denk ik ook, inderdaad voor later